jupyterlab_bigquery/jupyterlab_bigquery/list_items_handler/list_items_handler.py

```diff
@@ -0,0 +1,28 @@
+# Lint as: python3
+"""Request handler classes for the extensions."""
+
+from jupyterlab_bigquery.list_items_handler.create_handler import _handler
+from jupyterlab_bigquery.list_items_handler.service import BigQueryService
+
+SCOPE = ("https://www.googleapis.com/auth/cloud-platform",)
+
+list_items_handlers = {}
```

Comment:
If my understanding is correct, this variable should be part of the decorator logic and should be moved to be internal to the decorator file.

That way you wouldn't have to pass in the handlers dict. Let me know if I am not understand what you are trying to accomplish with the decorator functionality.